sdk/vaa/payloads.go

```diff
@@ -402,6 +428,47 @@ func (r BodyWormholeRelayerSetDefaultDeliveryProvider) Serialize() []byte {
 	return serializeBridgeGovernanceVaa(WormholeRelayerModuleStr, WormholeRelayerSetDefaultDeliveryProvider, r.ChainID, payload.Bytes())
 }
 
+func allZero(b []byte) bool {
+	for _, v := range b {
+		if v != 0 {
+			return false
+		}
+	}
+	return true
+}
+
+func (r BodyGeneralPurposeGovernanceEvm) Serialize() []byte {
+	payload := &bytes.Buffer{}
+	if !allZero(r.GovernanceContract[0:12]) {
+		panic("governance contract address must be 0-padded")
+	}
+
+	if !allZero(r.TargetContract[0:12]) {
+		panic("target contract address must be 0-padded")
+	}
+
+	payload.Write(r.GovernanceContract[12:])
+	payload.Write(r.TargetContract[12:])
+	// write payload len as uint16
+	if len(r.Payload) > math.MaxUint16 {
+		panic("payload too long")
+	}
+	MustWrite(payload, binary.BigEndian, uint16(len(r.Payload)))
+	payload.Write(r.Payload)
+	return serializeBridgeGovernanceVaa(GeneralPurposeGovernanceModuleStr, GovernanceAction(1), r.ChainID, payload.Bytes())
```

Comment:
instead of `GovernanceAction(1)`, should we add a type for `ActionGeneralPurposeGovernanceEvm`?